src/System.Management.Automation/FormatAndOutput/common/StringDecorated.cs

```diff
@@ -69,16 +69,7 @@ public string ToString(OutputRendering outputRendering)
                 return _text;
             }
 
-            if (outputRendering == OutputRendering.Automatic)
-            {
-                outputRendering = OutputRendering.Ansi;
-                if (PSStyle.Instance.OutputRendering == OutputRendering.PlainText)
-                {
-                    outputRendering = OutputRendering.PlainText;
-                }
-            }
-
-            if (outputRendering == OutputRendering.PlainText)
+            if (outputRendering == OutputRendering.PlainText || PSStyle.Instance.OutputRendering == OutputRendering.PlainText)
```

Comment:
I found a little problem to `StringDecorated.ToString` after removing `OutputRendering.Automatic`. Previously, we check `PSStyle.Instance.OutputRendering` only when the passed in value is `Automatic`, but now we do the check for any passed in value, which means the explicitly passed in value doesn't take precedence over `PSStyle.Instance.OutputRendering`. That seems not right.

How about changing to the following:
```
public override string ToString() => _isDecorated
    ? ToString(
           PSStyle.Instance.OutputRendering == OutputRendering.PlainText
               ? OutputRendering.PlainText
               : OutputRendering.Ansi)
    : _text;


public string ToString(OutputRendering outputRendering)
{
    if (!_isDecorated)
    {
        return _text;
    }

    return outputRendering == OutputRendering.PlainText ? PlainText : _text;
}

```

The above code maintains the existing behavior. However, when `Host` is passed in, `ANSI` version of the string will be returned, regardless of whether the output is going to the Host or not. That's a little broken given the definition of `OutputRendering.Host`.

Reply:
Ok, now I remember why I wanted `Automatic`.  As you explained, it's so that it can be passed to `ToString()`.  Perhaps it's better to keep `Automatic` even though it means `Host` for `$PSStyle.OutputRendering`.

Reply:
IMHO, it's compelling to add a new member `Automatic` for this `ToString` usage, especially it means different things in different scenarios. It would be hard to explain to the user.

Reply:
Yeah, just `ToString()` works like "automatic", so retricting `ToString(OutputRendering)` to just ANSI or PlainText makes sense.  I'll make that change.